diagnostic.md

```diff
@@ -44,9 +57,23 @@ How would we reference each of the following categories of elements?
 -   Only those elements belonging to both the `important` and `red` _classes_
 -   All `p` elements whose parent elements are `div`s
 
-<!-- your answer starts here -->
+```CSS
+.big{
 
-<!-- your answer ends here -->
+}
+
+#contentPane{
+
+}
+
+.important .red{
```

Comment:
This will select elements that have ```red``` as a class, that is inside of an element that has ```important``` as a class.  To select elements that have both ```important``` and ```red```, use ```.important.red```.